README.md

```diff
@@ -250,11 +272,11 @@ Get/set accessors do work as well
 class Person extends Model {
```

Comment:
Can't put a comment here, but also update the link in line 247 (@Unique decorator)

Reply:
Thanks fixed!